answer/src/main/java/org/exoplatform/platform/qa/ui/answer/pageobject/AnswerManagement.java

```diff
@@ -27,12 +32,7 @@ public AnswerManagement(TestBase testBase) {
    */
   public void goToAnswerQuestion(String question) {
     info("Go to answer a question");
-
-    if (evt.waitForAndGetElement(ELEMENT_QUESTION_LIST_ITEM.replace("$question", question), 5000, 0) != null)
-      evt.click(ELEMENT_QUESTION_LIST_ITEM.replace("$question", question));
-    evt.waitForAndGetElement(ELEMENT_QUESTION_SELECTED_ITEM.replace("$question", question));
-    evt.click(ELEMENT_ANSWER_BUTTON);
-    evt.waitForAndGetElement(ELEMENT_ANSWER_FORM);
+    $(byText(question)).parent().parent().parent().find(byText("Answer")).click();
```

Comment:
We should use the path instead of byText("Answer") to find the element to click IMO.